test/test_linalg.py

```diff
@@ -5610,8 +5610,11 @@ def call_torch_fn(*args, **kwargs):
             self.assertEqual([(2, 0, 0), (2, 0)], [A_LU.shape, pivots.shape])
 
     @dtypesIfCUDA(torch.cfloat, torch.cdouble,
-                  *get_all_fp_dtypes(include_half=not CUDA9, include_bfloat16=(CUDA11OrLater and SM53OrLater)))
-    @dtypes(*(set(get_all_dtypes()) - {torch.half, torch.bool}))
+                  *floating_types_and(
+                      *((torch.half, ) if not CUDA9 else ()),
+                      *((torch.bfloat16, ) if not (CUDA11OrLater and SM53OrLater) else ()))
```

Comment:
I think this `not` should be removed -- bfloat16 should be included if on CUDA11OrLater and SM53OrLater, right?

Reply:
Yes, you are right! I fixed it.